LCA_Analysis/LCA_data_preprocessing_2024_10_25.ipynb

```diff
@@ -0,0 +1,104 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv(\"data/poLCA_35128.csv\")\n",
```

Comment:
What is this dataset about, and where can we get it?

Reply:
This is the dataset is created by joining all patients from Ahmed and your sql scripts with the elixhauser index quan table. Since you are handling all the sql script, should I send the script to you?